package.json

```diff
@@ -3,7 +3,10 @@
 	"version": "7.7.2",
 	"description": "Get metadata about the active window (title, id, bounds, owner, URL, etc). Works on macOS, Linux, Windows.",
 	"license": "MIT",
-	"repository": "sindresorhus/active-win",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/sindresorhus/active-win.git"
+	},
```

Comment:
Don't make unrelated changes.

Reply:
It's related for the publishing of the binary at release. It's a requirement for the github action.